rust/lance-index/src/vector/flat/transform.rs

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright The Lance Authors
+
+use arrow_array::RecordBatch;
+use arrow_schema::Field;
+use lance_arrow::RecordBatchExt;
+use lance_core::Error;
+use snafu::location;
+use tracing::instrument;
+
+use crate::vector::transform::Transformer;
+
+use super::storage::FLAT_COLUMN;
+
+#[derive(Debug)]
+pub struct FlatTransformer {
```

Comment:
```suggestion

/// Renames the input column to FLAT_COLUMN
#[derive(Debug)]
pub struct FlatTransformer {
```

Does this transformer do anything else?  Or is it just renaming the column?